libs/dm/scripts/loadDM.ts

```diff
@@ -1,16 +1,13 @@
-import * as fs from 'fs';
 import { execSync } from "child_process";
+import * as fs from 'fs';
 import * as path from 'path';
 
 const projectDir = path.resolve(__dirname, "..")
 
 if (fs.existsSync(`${projectDir}/GenshinData`)) {
-  if (execSync(`cd ${projectDir}/GenshinData && git fetch --dry-run`).toString()) {
-    console.log("./GenshinData exists & needs update, updating repo...");
-    execSync(`cd ${projectDir}/GenshinData && git pull --depth 1`)
-  } else
-    console.log("./GenshinData exists & doesn't need update");
+  console.log(`${projectDir} exists. pulling repo...`);

```

Comment:
You didn't fetch before the reset. Also, may as well update the comment to say "fetching and reseting repo" rather than "pulling repo"